src/workerd/api/tests/js-rpc-test.js

```diff
@@ -1179,11 +1197,40 @@ export let crossContextSharingDoesntWork = {
 
 export let waitUntilWorks = {
   async test(controller, env, ctx) {
-    globalWaitUntilPromise = null;
-    await env.MyService.testWaitUntil();
+    // Tests ctx.waitUntil
+    {
+      globalWaitUntilPromise = null;
+      await env.MyService.testWaitUntil();
+
+      assert.strictEqual(globalWaitUntilPromise instanceof Promise, true);
+      await globalWaitUntilPromise;
+    }
+
+    // Tests `import { waitUntil } from 'cloudflare:workers` on DO
+    {
+      globalWaitUntilPromise = null;
+      await env.MyService.testImportedWaitUntil();
+
+      assert.strictEqual(globalWaitUntilPromise instanceof Promise, true);
+      await globalWaitUntilPromise;
+    }
 
-    assert.strictEqual(globalWaitUntilPromise instanceof Promise, true);
-    await globalWaitUntilPromise;
+    // Tests `import { waitUntil } from 'cloudflare:workers` directly
```

Comment:
I'm not sure what's the value of this test compared to the previous block. It doesn't look like this block actually testes anything as nobody actually waits on the `globalWaitUntilPromise` to make sure it resolves. But the previous block already covers that anyway, so I think this block can simply be deleted?